test/fixtures/users.yml

```diff
@@ -4,6 +4,7 @@ family_admin:
   last_name: Dylan
   email: bob@bobdylan.com
   password_digest: <%= BCrypt::Password.create('password') %>
+  role: admin
```

Comment:
I just wonder why whoever created this test fixture never bothered to specify the admin role?